### PR TITLE
Bench the sad function for 8 bit and 10 bit

### PR DESCRIPTION
**Pull request by magwyz**
They have different paths and ASM functions.

### CODE REVIEW

**Comment by magwyz** [comment]
Maybe we should have two tests instead?

File: benches/me.rs [hunk 1]
```diff
@@ -33,13 +33,13 @@ fn new_plane<T: Pixel>(ra: &mut ChaChaRng, width: usize, height: usize) -> Plane
   p
 }
 
-fn bench_get_sad(b: &mut Bencher, bs: &BlockSize) {
+fn bench_get_sad(b: &mut Bencher, input_type: &(BlockSize, usize)) {
+  let (bs, bit_depth) = input_type;
```
Comment (shssoichiro) [inline benches/me.rs]: I'd prefer these as two parameters to the function, rather than as one `input_type` tuple.

File: benches/me.rs [hunk 2]
```diff
@@ -33,13 +33,13 @@ fn new_plane<T: Pixel>(ra: &mut ChaChaRng, width: usize, height: usize) -> Plane
   p
 }
 
-fn bench_get_sad(b: &mut Bencher, bs: &BlockSize) {
+fn bench_get_sad(b: &mut Bencher, input_type: &(BlockSize, usize)) {
```
Comment (rom1v) [inline benches/me.rs]: The `(BlockSize, usize)` could be passed by value, there is no need for reference here.

File: benches/me.rs (hunk 1 above)
Comment (magwyz) [reply benches/me.rs]: I think we are indeed limited by the `bench_function_over_inputs` function that takes a single `inputs` array as parameter.

**Comment by rzumer** [comment]
> 
> 
> Maybe we should have two tests instead?

I would lean towards yes, since it would make it easier to identify whether a regression affects one path or the other (or both) without modifying the test.

**Pull request merged**